src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -67,8 +67,8 @@ int LFUDAPolicy::init(CephContext *cct, const DoutPrefixProvider* dpp, asio::io_
 
   driver = _driver;
   if (dpp->get_cct()->_conf->d4n_writecache_enabled) {
+    quit = false;
```

Comment:
Without explicitly resetting the `quit` value here, it remains true in later tests and as a result, the cleaning method does not execute.

Reply:
ok